api/resources/views/layouts/app.blade.php

```diff
@@ -55,6 +55,7 @@
                                 </a>
 
                                 <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown">
+                                    <a class="dropdown-item" href="{{ route('posts.index') }}">Manage Posts<span class="sr-only">(user posts)</span></a>
```

Comment:
I think this should be moved. It's really unintuitive to have to find a button hidden under a profile dropdown. It should probably be moved to the profile page (which I will gladly do when that's done and merged, if that's fine with you) and a "create post" button should be inserted in the home page, because you can only read posts there and it's once again hard to find where to create a post for yourself.

Reply:
Manage Posts seems as an intuitive way of saying to a user that he can do everything with his posts there. I think it would be better to leave it as is, but yeah, let's move it out of the dropdown. Will you do that with your changes?

Reply:
Yes, I completely agree with you on having a dedicated button or something for a user to access all his post, I just think it would fit better in his profile or his settings.